pkg/metrics/metrics.go

```diff
@@ -261,6 +280,29 @@ func (opMgr *operationMetricsManager) init() {
 		[]string{labelDriverName, labelOperationName, labelSnapshotType, labelOperationStatus},
 	)
 	opMgr.registry.MustRegister(opMgr.opLatencyMetrics)
+	opMgr.opInFlight = k8smetrics.NewGauge(
+		&k8smetrics.GaugeOpts{
+			Subsystem: subSystem,
+			Name:      operationInFlightName,
+			Help:      operationInFlightHelpMsg,
+		},
```

Comment:
it might be helpful to also add labels like DriverName

Reply:
The goal of this metric, per #434, is mainly for us to see if any leaks are happening. The metric counts all operations in transit regardless of driver name/labels.

There are some cases where we won't have driver name. i.e. in the scheduled go-routine and in DropOperation.